### DISCUSSION TITLE
A leading space in only-url log line of action corrupts the line

### DISCUSSION THREAD

**Discussion by Felixoid** [community/community#24031] [Other Feature Feedback, Questions, & Ideas]
<p>Hello, here’s a very strange issue with showing logs. I can’t decide where to report it but here.</p>
<p>Here’s the <a href="https://github.com/ClickHouse/ClickHouse/runs/7405843121?check_suite_focus=true#step:5:29" rel="noopener nofollow ugc">link</a> to a broken example. Originally, it’s the following line in raw logs:</p>
<pre><code class="lang-auto">2022-07-19T09:03:02.0380304Z  https://github.com/ClickHouse/ClickHouse/pull/39283
</code></pre>
<p>But in the action logs it’s shown as following<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/9/e/9e80fec86a3bd1547411b4c547179a012064529c.png" data-download-href="/uploads/short-url/mCbOiUbZuc7zhXoPcIpMU0ZNegY.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/3025537/181080862-fc42809e-db37-421b-91c9-d6bc0c250941.png" alt="image" data-base62-sha1="mCbOiUbZuc7zhXoPcIpMU0ZNegY" width="690" height="423" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/9/e/9e80fec86a3bd1547411b4c547179a012064529c_2_690x423.png, https://aws1.discourse-cdn.com/github/original/3X/9/e/9e80fec86a3bd1547411b4c547179a012064529c.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/9/e/9e80fec86a3bd1547411b4c547179a012064529c.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/9/e/9e80fec86a3bd1547411b4c547179a012064529c_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1011×620 70.9 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>Urls in the link are the correct though. Only displayed text is cut by the last charecter</p>

**Comment by jsoref** [comment, marked as answer]
<p>The best forum for this feedback is here:</p><aside class="onebox allowlistedgeneric" data-onebox-src="https://github.com/github-community/community/discussions/categories/actions-and-packages">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">

      <a href="https://github.com/github-community/community/discussions/categories/actions-and-packages" target="_blank" rel="noopener">GitHub</a>
  </header>

  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/345;"><img src="https://user-images.githubusercontent.com/2119212/181080871-79c10f08-31db-4566-b68a-28e071446d00.png" class="thumbnail" width="690" height="345" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/d/7/d7a1ce34ec9399d4fef818780831dd1730cafba3_2_690x345.png, https://aws1.discourse-cdn.com/github/optimized/3X/d/7/d7a1ce34ec9399d4fef818780831dd1730cafba3_2_1035x517.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/d/7/d7a1ce34ec9399d4fef818780831dd1730cafba3.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/d/7/d7a1ce34ec9399d4fef818780831dd1730cafba3_2_10x10.png"></div>

<h3><a href="https://github.com/github-community/community/discussions/categories/actions-and-packages" target="_blank" rel="noopener">Discussions · github-community/community</a></h3>

  <p>Public feedback discussions for: GitHub for Mobile, GitHub Discussions, GitHub Codespaces, GitHub Sponsors, GitHub Issues and more! - Discussions · github-community/community</p>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

<p>Alternatively, you could file a bug via:</p><aside class="onebox allowlistedgeneric" data-onebox-src="https://support.github.com/">
  <header class="source">
      <img src="https://user-images.githubusercontent.com/2119212/181080873-02d7ea3d-f7ef-4133-90c4-28405b2ed469.png" class="site-icon" width="32" height="32">

      <a href="https://support.github.com/" target="_blank" rel="noopener">support.github.com</a>
  </header>

  <article class="onebox-body">
    

<h3><a href="https://support.github.com/" target="_blank" rel="noopener">GitHub Support</a></h3>

  <p>GitHub Support is here to help. Learn about GitHub products, browse our helpful resources, and contact support with your questions.</p>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>
